src/OpenTelemetry/Metrics/MeterProviderBuilderExtensions.cs

```diff
@@ -71,6 +76,16 @@ public static MeterProviderBuilder AddView(this MeterProviderBuilder meterProvid
         /// <remarks>See View specification here : https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view.</remarks>
         public static MeterProviderBuilder AddView(this MeterProviderBuilder meterProviderBuilder, string instrumentName, MetricStreamConfiguration metricStreamConfiguration)
         {
+            if (metricStreamConfiguration == null)
+            {
+                throw new ArgumentNullException($"Metric stream configuration cannot be null.", nameof(metricStreamConfiguration.Name));
```

Comment:
Sorry, I am new here but this line caught my eye. Shouldn't this be `throw new ArgumentNullException($"Metric stream configuration cannot be null.", nameof(metricStreamConfiguration));` ?

Reply:
Yes, you're right @randomnoise :). Fixed and thanks!